specs/interop/predeploys.md

```diff
@@ -73,47 +72,30 @@ an update to the `L1Block`, `OptimismMintableERC20Factory` and `L2StandardBridge
 | -------- | -------------------------------------------- |
 | Address  | `0x4200000000000000000000000000000000000022` |
 
-The `CrossL2Inbox` is responsible for executing a cross chain message on the destination chain.
-It is permissionless to execute a cross chain message on behalf of any user.
+The `CrossL2Inbox` is the system predeploy for cross chain messaging. Anyone can trigger the execution or validation
+of cross chain messages, on behalf of any user.
 
 To ensure safety of the protocol, the [Message Invariants](./messaging.md#messaging-invariants) must be enforced.
 
-### Message execution arguments
-
-The following fields are required for executing a cross chain message:
-
 [message payload]: ./messaging.md#message-payload
 [`Identifier`]: ./messaging.md#message-identifier
 
+### Functions
+
+#### executeMessage
+
+Executes a cross chain message and performs a `CALL` with the payload to the provided target address, allowing
+introspection of the data.
+Signals the transaction has a cross chain message to validate by emitting the `ExecuteMessage` event.
+
+The following fields are required for executing a cross chain message:
+
 | Name      | Type         | Description                                             |
 | --------- | ------------ | ------------------------------------------------------- |
 | `_msg`    | `bytes`      | The [message payload], matching the initiating message. |
 | `_id`     | `Identifier` | A [`Identifier`] pointing to the initiating message.    |
 | `_target` | `address`    | Account that is called with `_msg`.                     |
 
-#### `_msg`
```

Comment:
The following sections read very redundantly to the table above and the pages they link to. 

What is important to keep however is the elaboration on `_target`